src/shared/udev-util.c

```diff
@@ -82,6 +84,13 @@ int udev_parse_config_full(
                                    "failed to set parse event_timeout=%s, ignoring: %m", event_timeout);
         }
 
+        if (ret_exit_timeout_usec && exit_timeout) {
+                r = parse_sec(exit_timeout, ret_exit_timeout_usec);
```

Comment:
`parse_sec_fix_0()` would be more appropriate I think since "0" would be equivalent to "infinity"

Reply:
udevd uses `parse_sec()` for `event_timeout`, not using the "0 == infinity" convention. I'd prefer to be consistent here. We can allow "0 == infinity" for all timeouts in udevd, but that would be a separate PR I think.